charts/handbrake/1.0.0/test_values.yaml

```diff
@@ -77,37 +77,43 @@ additionalVolumes: []
 #           values:
 #           - opticaldrive-controller
 
-#appVolumeMounts:
-#  config:
-#    enabled: true
-#    emptyDir: false
-#    datasetName: "config"
-#    mountPath: "/config"
-#    hostPathEnabled: false
-#  storage:
-#    enabled: true
-#    emptyDir: false
-#    datasetName: "storage"
-#    mountPath: "/storage"
-#    hostPathEnabled: false
-#  watch:
-#    enabled: true
-#    emptyDir: false
-#    datasetName: "watch"
-#    mountPath: "/watch"
-#    hostPathEnabled: false
-#  output:
-#    enabled: true
-#    emptyDir: false
-#    datasetName: "output"
-#    mountPath: "/output"
-#    hostPathEnabled: false
-#  opticaldrive:
-#    enabled: true
-#    emptyDir: false
-#    mountPath: "/dev/sr0"
-#    hostPathEnabled: true
-#    hostPath: "/dev/sr0"
+appVolumeMounts:
+ config:
+   enabled: true
+   emptyDir: false
```

Comment:
needs to be true on all of them for test_values

Reply:
(otherwise it won't pass the tests)

Reply:
Still all needs to be true in test_values.yaml, even though this comment is closed.
Not passing CI because of it.

Reply:
whoops, didn't saved the file before